src/components/PostComment.vue

```diff
@@ -1,134 +1,123 @@
 <template>
-  <div class="post-comment">
-    <div class="comment-metadata">
-      <img :src="authorProfilePictureUrl" class="comment-author-profile-picture"/>
-      <div class="comment-author"> {{ author }} </div>
-      <div class="comment-time"> {{ date }} </div>
-      <div class="dropdown is-right is-hoverable alignright">
-        <div class="dropdown-trigger">
-          <button class="button no-border" aria-haspopup="true" aria-controls="dropdownMenu">
-            <span class="icon">
-              <i class="material-icons">more_vert</i>
-            </span>
-          </button>
-        </div>
-        <div class="dropdown-menu no-border" id="dropdownMenu" role="menu">
-          <div class="dropdown-content">
-            <div class="dropdown-item">
-              <!-- <a v-if="userNickname === author" href="#" class="dropdown-item">
-                수정
-              </a> -->
-              <a
-                v-if="userNickname === author"
-                @click="deleteComment"
-                class="dropdown-item"
-              >
-                {{ $t('delete') }}
-              </a>
-              <a v-else href="#" class="dropdown-item">
-                {{ $t('report') }}
-              </a>
+  <div class="comment-wrapper">
+    <div class="comment" :class="{'comment--recomment': isRecomment}" v-if="!isEditing">
```

Comment:
I think _replyComment(or reply-comment or others)_ is easier to understand than _recomment_.